src/main/data/TwoBit.js

```diff
@@ -124,6 +136,32 @@ function markUnknownDNA(basePairs: Array<string>, dnaStartIndex: number, sequenc
   return basePairs;
 }
 
+/**
+ * Try getting a bigger chunk of the remote file
+ * until the Range Error is resolved. This is useful when we need to
+ * parse the header, but when we don't know the size of the header up front.
+ * If the intial request returns an incomplete header and hence the
+ * parsing fails, we next try doubling the requested size.
+ * The size of the request is capped with `untilSize` so that
+ * we don't start asking for MBs of data for no use.
+ * Instead we we throw an error if we reach the cap,
+ * potentially meaning a corrupted 2bit file.
+*/
+function retryRemoteGet(remoteFile: RemoteFile, start: number, size: number, untilSize: number, promisedFunc: Function) {
+  return remoteFile.getBytes(start, size).then(promisedFunc).catch(error => {
+    if(error.constructor == RangeError) {
```

Comment:
Could be nice to catch that error within the parser and throw a more descriptive error; would that be reasonable for this PR? The `RangeError` in the context of this function is pretty confusing (I was wondering why we'd catch a `RangeError` and then try to get a longer chunk).